pages/getting-started.html

```diff
@@ -0,0 +1,45 @@
+---
+permalink: /getting-started/
+layout: styleguide
+title: Getting started
+lead: Intro text on what is included in this section and how to use it. No more than one or two sentences.
+---
+
+<h3 class="usa-heading">For developers</h3>
+
+<p><b>HTML</b> is built with progressive enhancement. All users get critical information and experiences. New browsers get the prettiest experiences, older ones get less pretty, but usable ones. If JavaScript fails, users will still get a robust HTML foundation.</p>
```

Comment:
Shouldn't this use `<strong>` rather the `<b>`.

"According to the HTML 5 specification, the <b> tag should be used as a LAST resort when no other tag is more appropriate." - w3c